### ISSUE TITLE
Improve ease of use of gridTestUtils.spec.js

### ISSUE THREAD

**Issue by mscharp** [angular-ui/ng-grid#2692]
I've been using the gridTestUtils.spec.js file to create e2e tests.  After creating a handful of tests, and given that I am testing a single grid per test spec, it became increasingly annoying to have to specify the gridId for each call to the util.  In a before block, it would be nice to specify the grid Id to use for all of the tests.  I propose adding two methods:
```javascript
setGridId: function( gridId ) {
   _gridId = gridId;
}
getGrid: function( gridId ) {
    return element( by.id( gridId ? gridId : _gridId ) );
}
```
And refactoring all methods to use getGrid() and to change the method signatures by having gridId as the last, and therefore, optional parameter.

This would require changing all of the e2e tests, but gives testers the ability to specify the gridId only once as well as still enabling them to override ( or never set ) the global gridId.

**Comment by mscharp** [comment]
I've actually changed this util for my current project.  Thought about forking and submitting a PR, but thought I'd see if anyone else thought it would be beneficial before I went through the work to do that.

**Comment by PaulL1** [comment]
I think it would be useful, and a PR would be fine.  I'm excited that someone is actually using the test utils!!

**Comment by mscharp** [comment]
Ok, after more e2e testing, I found that this wan't a great approach due to jasmine framework's limitations.

Essentially, what I wanted to do was set the gridId once for all of the tests I would be running in a single spec.  However, with the way jasmine (1.X) works in regard to "spec" setup and executing the describe blocks of multiple spec files at (seemingly) random times, I couldn't sync setting the gridId with the actual tests being executed.  Jasmine 2.1 fixes this with a beforeAll function, and while the proposed setGridId functionality wasn't technically incompatible with jasmine 1.X, it wasn't really helpful either.

So, in lieu of this method, I've started setting a gridId constant in each spec file and then passing in that constant to each of the gridTestUtils methods that require it.  In this way, if I need to change the grid to be tested in a spec file I only need to change it in one place.

```
var gridTestUtils = require( '../lib/gridTestUtils.spec.js' );
describe( 'something', function ()
{
	var gridId = 'myGrid';
        it('should do something', function() {
             gridTestUtils.expectRow( gridId, 'asdf');
	});
}
```
So, all that to say, I recommend not doing this enhancement!  :)
